### PR TITLE
[MVC-988]  - Refactor Job Posting

### PR DESCRIPTION
**Pull request by nashthecoder**
[MVC-988]  - Refactor Job Posting
- Reduced card size (Only show position and location) 
-  text under Open positions
- Moved Open Application to last 
- When hovering the position it shows the description 

Challenge 
1. Flickering on hover of title  see video  - researching  on this , may need help to fix.
https://user-images.githubusercontent.com/15178823/119821641-768db800-befb-11eb-984a-72147d25a95a.mov

[MVC-988]: https://triggerise.atlassian.net/browse/MVC-988

### CODE REVIEW

**Review by coertg — APPROVED** [review]

**Comment by nashthecoder** [comment]
Fixed flickering on hover using css. 
https://user-images.githubusercontent.com/15178823/119848695-b2814700-bf14-11eb-8851-e6074c119469.mov

- Not sure how to test for mobile from localhost:4000. @Triggerise/todo any pointers

**Comment by tiagovarela** [comment]
> * Not sure how to test for mobile from localhost:4000. @Triggerise/todo any pointers

@nashthecoder you can use the Chrome DevTools to test using a mobile device view: https://developer.chrome.com/docs/devtools/device-mode/

**Comment by nashthecoder** [comment]
> > * Not sure how to test for mobile from localhost:4000. @Triggerise/todo any pointers
> 
> @nashthecoder you can use the Chrome DevTools to test using a mobile device view: https://developer.chrome.com/docs/devtools/device-mode/

I would like to test the touch/tap  feature for mobile.

**Comment by tiagovarela** [comment]
> > > * Not sure how to test for mobile from localhost:4000. @Triggerise/todo any pointers
> > 
> > 
> > @nashthecoder you can use the Chrome DevTools to test using a mobile device view: https://developer.chrome.com/docs/devtools/device-mode/
> 
> I would like to test the touch/tap feature for mobile.

The `Device Type` should allow you to test `touch` events: https://developer.chrome.com/docs/devtools/device-mode/#type

**Comment by nashthecoder** [comment]
> > > > * Not sure how to test for mobile from localhost:4000. @Triggerise/todo any pointers
> > > 
> > > 
> > > @nashthecoder you can use the Chrome DevTools to test using a mobile device view: https://developer.chrome.com/docs/devtools/device-mode/
> > 
> > 
> > I would like to test the touch/tap feature for mobile.
> 
> The `Device Type` should allow you to test `touch` events: https://developer.chrome.com/docs/devtools/device-mode/#type

Thank you for the tip - learnt something new today!

**Comment by nashthecoder** [comment]
Tested - Touch working on mobile version.

File: _includes/jobs.html [hunk 1]
```diff
@@ -5,14 +5,21 @@ <h1>Open positions</h1>
...
+              <h3 class="location">{{job.Location}}</h3>
+              <div class="uk-hidden-hover uk-iconnav">
+                <p>{{job.['Job Description']}}</p>
+                <small><a href="{{job.Link}}">Read full description</a></small>
+              </div>
+            </div>
+        </div>
       {% endfor %}
+      </div>
+
```
Comment (raguiar9080) [inline _includes/jobs.html]: ✂️ empty line

**Review by raguiar9080 — APPROVED** [review]
Nice job 👍

File: _includes/jobs.html (hunk 1 above)
Comment (nashthecoder) [reply _includes/jobs.html]: Removed the space.

**Pull request merged**